pkg/metrics/kubelet_metrics.go

```diff
@@ -96,6 +96,30 @@ var KnownKubeletMetrics = map[string][]string{
 	"rest_client_request_status_codes":                       {"code", "host", "method"},
 }
 
+var InterestingKubeletMetricsForE2E = []string{
```

Comment:
Yeah - I agree that it's kind of problematic. But I think I have better idea now - I'll update this PR soon.

Basically I need to have 'PrintHumanReadable' that does not spill out all metrics (especially true for Kubelet, which has TONS of them), as it would be a complete mess.